src/components/Checkbox/Checkbox.e2e.ts

```diff
@@ -1,34 +1,58 @@
 import * as eyes from 'eyes.it';
-import {getStoryUrl} from 'wix-ui-test-utils/protractor';
+import {getStoryUrl, scrollToElement, mouseLeave} from 'wix-ui-test-utils/protractor';
 import {checkboxTestkitFactory} from 'wix-ui-core/dist/src/testkit/protractor';
 import {browser} from 'protractor';
+import autoExampleDriver = require('wix-storybook-utils/AutoExampleDriver');
 
 describe('Checkbox', () => {
   const storyUrl = getStoryUrl('Components', 'Checkbox');
   const dataHook = 'storybook-checkbox';
 
   beforeEach(() => browser.get(storyUrl));
+  afterEach(async () => {
+    await autoExampleDriver.reset();
+  });
 
-  eyes.it('should render', () => {
-    const driver = checkboxTestkitFactory({dataHook});
-
-    expect(driver.element()).toBeDefined();
+  describe('not hovered', () => {
+    eyes.it('should be unchecked', () => {
+      const driver = checkboxTestkitFactory({dataHook});
+      expect(driver.isChecked()).toBeFalsy();
+    });
+    
+    eyes.it('should be checked', async () => {
+      autoExampleDriver.setProps({checked: true});
+      const driver = checkboxTestkitFactory({dataHook});
+      await mouseLeave()
```

Comment:
why do you do mouseLeave?

Reply:
To make sure that mouse is not hovering the component from previous test.